.github/workflows/ci-pytorch_test-full.yml

```diff
@@ -143,7 +147,7 @@ jobs:
       run: |
         # adjust versions according installed Torch version
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/examples.txt
-        pip install -r requirements/pytorch/examples.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html --upgrade
+        pip install $PIP_FLAG -r requirements/pytorch/examples.txt --find-links "$TORCH_URL --upgrade
```

Comment:
so what we can do here is reinstall the all above to reduce installing different versions to be not sliding with example requirements...